docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_testing-in-testing-farm.adoc

```diff
@@ -0,0 +1,51 @@
+= Adding an integration test
+
+link:https://docs.testing-farm.io/[Testing Farm] is a third-party test service that can be used as a test execution back-end for {ProductName}. This document guides you through an example of how to trigger Testing Farm tests from your {ProductName} *Integration tests*.
+
+.Prerequisites
+
+* You have created an application in {ProductName}.
+* You have a Testing Farm API key, which can be acquired by following the link:https://docs.testing-farm.io/Testing%20Farm/0.1/onboarding.html[Testing Farm onboarding guide].
+* You have have or are ready to create a git repository with link:https://fmf.readthedocs.io/[fmf] files suitable for use with the link:https://tmt.readthedocs.io/[tmt] testing tool. This is the testing technology used by Testing Farm under the hood. If you don't have a fmf tests already prepared, you can use our tmt link:https://github.com/ralphbean/tmt-hello-world[hello world repository].
```

Comment:
While I personally like wording like "under the hood," in technical docs, Red Hat/IBM discourage us from using language like that to ensure we don't lose readers, especially for whom English is a second language.